solc/CommandLineInterface.cpp

```diff
@@ -225,6 +225,30 @@ static bool needsHumanTargetedStdout(po::variables_map const& _args)
 	return false;
 }
 
+static bool checkOutputComponent(po::variables_map const& _args)
```

Comment:
I think this could be used by the `needsHumanTargetedStdout` helper?

Reply:
Both have different argument in for loop, so not sure if we can really do it.

Reply:
The only difference seems that `g_argAst` is only in `checkOutputComponent`.